packages/cashier/src/pages/account-transfer/account-transfer-form/account-transfer-form-side-note.tsx

```diff
@@ -4,14 +4,13 @@ import { GetLimits } from '@deriv/api-types';
 import { Text } from '@deriv/components';
 import { getCurrencyDisplayCode, getPlatformSettings } from '@deriv/shared';
 
-type TAccountTransferNoteProps = {
+export type TAccountTransferNoteProps = {
```

Comment:
Do we need to export this? as far as I checked we don't use it anywhere! maybe I missed somewhere :thinking:

Reply:
I use it in test file to mock the props 😄

Reply:
I saw that hes using this in the test file itself, maybe instead we can separate it out to a types file?

Reply:
How about using this approach instead of exporting the type?

```
React.ComponentProps<typeof AccountTransferNote>
```